staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/conversion.go

```diff
@@ -27,15 +27,21 @@ import (
 func addConversionFuncs(scheme *runtime.Scheme) error {
 	// Add non-generated conversion functions
 	err := scheme.AddConversionFuncs(
+		Convert_v1beta1_JSONSchemaProps_To_apiextensions_JSONSchemaProps,
 		Convert_apiextensions_JSON_To_v1beta1_JSON,
-		Convert_v1beta_JSON_To_apiextensions_JSON,
+		Convert_v1beta1_JSON_To_apiextensions_JSON,
 	)
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
+// Convert_v1beta1_JSONSchemaProps_To_apiextensions_JSONSchemaProps is an autogenerated conversion function.
+func Convert_v1beta1_JSONSchemaProps_To_apiextensions_JSONSchemaProps(in *JSONSchemaProps, out *apiextensions.JSONSchemaProps, s conversion.Scope) error {
```

Comment:
this should be automatic, shouldn't it? Would not put that here.

Reply:
yeah, will remove this.

Reply:
I think the error was only in the typo, that should fix the conversion as a whole.